src/Scroller.js

```diff
@@ -128,6 +130,8 @@ export default class CalendarScroller extends Component {
 
   // Scroll to given date, and check against min and max date if available.
   scrollToDate = (date) => {
+    if (!this.props.scrollToOnSetSelectedDate) return;
```

Comment:
I think you should move this code into condition "if (minDate && targetDate.isBefore(minDate, "day")) {"